src/auctions.test.ts

```diff
@@ -1,14 +1,59 @@
 import {
-  calculateMinimumAuctionBid,
+  calculateAuctionPriceForBlock,
   getAuctionPricesForInterval,
   getEndTimestampForAuction,
 } from './auctions';
-import { SECONDS_IN_A_YEAR } from './constants';
+import { AUCTION_SETTINGS, SECONDS_IN_A_YEAR } from './constants';
 import { AuctionData, BlockTimestamp } from './types';
 import { BlockHeight } from './types';
 
 describe('Auction util functions', () => {
-  describe('calculateMinimumAuctionBid function', () => {
+  describe('validate AUCTION_SETTINGS used in the contract', () => {
+    const basePrice = 30;
+    const allowedThreshold = 0.05; // prices must be within 5% of the expected value
```

Comment:
How did you arrive at 5% for this epsilon?

Reply:
Eh mostly giving us ability to make small changes with confidence we're not totally busting the shape. 5% seemed reasonable but we can tighten even lower (or set it against exact values)